config/locales/client.en.yml

```diff
@@ -339,8 +339,10 @@ en:
       copied: "copied!"
 
     drafts:
-      label: "Drafts"
-      label_with_count: "Drafts (%{count})"
+      label:
```

Comment:
I’m not sure of the impact of this change on all the other locales. I hope not to break them all :sweat_smile:

Reply:
See https://github.com/discourse/discourse/pull/14501#discussion_r722258300

It should be fine to add the `one` and `other` keys to `label_with_count`. I'm pretty sure the `I18n` implementation falls back to English when it expects a Hash, but gets a string for other locales. You can always give it a try by changing the locale in your dev environment. Translations are updated every Tuesday which will remove the old translations for other locales.

Reply:
I’ll revert the change as this refactor doesn’t make sense anymore (sadly).